github/orgs_audit_log.go

```diff
@@ -69,16 +70,14 @@ type AuditEntry struct {
 	TargetLogin          *string    `json:"target_login,omitempty"`
 	TargetVersion        *string    `json:"target_version,omitempty"`
 	Team                 *string    `json:"team,omitempty"`
+	Timestamp            *int64     `json:"@timestamp,omitempty"`
+	TransportName        *string    `json:"transport_protocol_name,omitempty"`
```

Comment:
To minimize confusion and maximize consistency, could we please change this from `TransportName` to `TransportProtocolName`?